pkg/state/chart_dependency.go

```diff
@@ -404,6 +409,25 @@ func (m *chartDependencyManager) Resolve(unresolved *UnresolvedDependencies) (*R
 		return nil, false, err
 	}
 
+	// Make sure go run main.go works.
+	if version.Version != "" || lockedReqs.Version != "" {
```

Comment:
Shouldn't this better be `version.Version != "" && lockedReqs.Version != ""`?

Otherwise L414 seems to fail due to an empty version when the lock file doesn't contain any version number, which is very likely `helmfile.lock` is generated by an older version of helmfile that doesn't include this feature.

Reply:
Yes, you are absolutely correct.